pkgs/by-name/di/dirbuster/package.nix

```diff
@@ -0,0 +1,74 @@
+{ lib
+, stdenvNoCC
+, fetchurl
+, makeWrapper
```

Comment:
```suggestion
, makeBinaryWrapper
```

Use `makeBinaryWrapper` for a better compatibility with Darwin architectures.

Reply:
`makeBinaryWrapper` results in the process not starting 🤔

Reply:
Oh. OK let's leave it like that then. Weird, but ok.

Reply:
it was due to the `binutils` dependency (for `strings`). fixed by dropping `stdenvNoCC`.

Reply:
TIL ! Thank you !